include/beast/websocket/detail/mask.hpp

```diff
@@ -60,11 +60,17 @@ void
 maskgen_t<_>::rekey()
 {
     std::random_device rng;
+#if 0
     std::array<std::uint32_t, 32> e;
```

Comment:
This might work:

    std::uint32_t e[32];
    std::generate(std::begin(e), std::end(e), std::ref(rng));
    std::seed_seq ss(std::begin(e), std::end(e));